bip-0301.mediawiki

```diff
@@ -0,0 +1,234 @@
+<pre>
+
```

Comment:
Please check the preamble is formatted correctly. (There shouldn't be blank lines here.)

README will also need to be updated.